lib/indexer/document_preparer.rb

```diff
@@ -27,7 +38,7 @@ def prepare_popularity_field(doc_hash, popularities)
     end
 
     def prepare_tags_field(doc_hash)
-      Indexer::TagLookup.new.prepare_tags(doc_hash)
```

Comment:
Actually chucked that in temporarily to make stubbing the interface easier in some other test, and just left it in because I thought it was fine. Not terribly fussed - will revert this and update the  corresponding test setup.